README.md

```diff
@@ -1225,9 +1225,9 @@ IEEE Int. Conf. Computer Vision (ICCV), 2021. [Project page](http://vis-www.cs.u
 
 <a name="visual-servoing"></a>
 ### Visual Servoing
-- <a name="Muthusamy20arxiv"></a>Muthusamy, R., Ayyad, A., Halwani, M., Zweiri, Y., Gan, D., Seneviratne, L.,  
-*[Neuromorphic Eye-in-Hand Visual Servoing](https://arxiv.org/pdf/2004.07398)*,  
-arXiv, 2020.
+- <a name="Muthusamy20ieeeaccess"></a>Muthusamy, R., Ayyad, A., Halwani, M., Swart, D., Gan, D., Seneviratne, L., Zweiri, Y.,   
```

Comment:
key: Muthusamy21access
official link: https://doi.org/10.1109/ACCESS.2021.3071261  (instead of ieeexplore...)